src/main/resources/locales/cy/party-name-search-results.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Canlyniad y chwilio",
-    "resultsCount": " Wedi dod i hyd i ganlyniad(au)",
+    "resultsCount": "Wedi canfod o ganlyniadau",
```

Comment:
Not sure if this is correct. The Welsh text in the ticket is as below, so you number of results needs to be injected between 'canfod' and 'o'. You can add a placeholer text then in the njk file replace the placeholder with the actual count. 
Wedi canfod [X] o ganlyniadau

Reply:
Committed changes